### PR TITLE
Update use-notifications-and-policy-tips.md

### PR DESCRIPTION
**Pull request by ybhargav1995**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/use-notifications-and-policy-tips.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ For each rule in a DLP policy, you can:
 - Customize the text that's included in the notification by using HTML or tokens. See the section below for more information.
 
 > [!NOTE]
-> Email notifications can be sent only to individual recipients—not groups or distribution lists. Only new content will trigger an email notification. Editing existing content will trigger policy tips, but not an email notification.
+> Email notifications can be sent only to individual recipients—not groups or distribution lists. Only new content will trigger an email notification. Editing existing content will trigger policy tips, but not an email notification. If an email from a user external to the organization that has set the rule, they wouldnt receive any notification as these notifications only go to internal users.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/use-notifications-and-policy-tips.md]:
```suggestion
> Email notifications can be sent only to individual recipients—not groups or distribution lists. Only new content will trigger an email notification. Editing existing content will trigger policy tips, but not an email notification. If an email is sent from a user external to the organization that has set the rule, they will not receive any notification as these notifications only go to internal users.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ybhargav1995, kindly commit the suggested changes. Many thanks.

**Comment by ybhargav1995** [comment]
@chrfox @JohanFreelancer9 can you please merge this

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**